csrc/id_model/loop_promotion.cpp

```diff
@@ -682,6 +676,16 @@ IterDomain* LoopPromotionMapBuilder::findPromotionOfLoopGroup(
       continue;
     }
 
+    // If this domain is a view rfactor domain and a terminal domain,
+    // it is guaranteed to represent this loop group because all the
+    // domains merged into this loop_id must be non-broadcast
+    // domains. A concrete example can be found in test
```

Comment:
> because all the domains merged into this loop_id must be non-broadcast domains

Could you elaborate more in this comment? 

> when we do reshape transformations, all of the broadcast domains in the input tensor are first squeezed

Reply:
When reshaping a tensor like `{10, 1, 4}` to `{40}`, the input tensor is first squeezed to `{10, 4}`, and the actual reshape op would be `{10, 4}` to `{40}`, so we would never create a view rfactor domain of `{10*1*4}`.

Reply:
I understand. I meant that we should add this description to the code.